velox/exec/fuzzer/FuzzerUtil.cpp

```diff
@@ -50,6 +51,16 @@ std::vector<Split> makeSplits(
   return splits;
 }
 
+std::vector<Split> makeSplits(const std::string& directory) {
```

Comment:
Why not you pass a list of files to make splits? Thanks!

Reply:
So when we have partitioned table, the directory would look like
/p0=0/p1=0/file1
/p0=1/p1-0/file2
...

In such cases, we need to be able to parse out partition information and put it in the split schema